launchers/data-loader-cli/src/test/resources/assets.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"asset":{"properties":{"asset:prop:name":"test-asset-0","asset:prop:version":"1.0","asset:prop:id":"test-asset-0"}},"dataAddress":{"properties":{"someprop":"someval","type":"test-dataaddress-0"},"type":"test-dataaddress-0"}},{"asset":{"properties":{"asset:prop:name":"test-asset-1","asset:prop:version":"1.0","asset:prop:id":"test-asset-1"}},"dataAddress":{"properties":{"someprop":"someval","type":"test-dataaddress-1"},"type":"test-dataaddress-1"}},{"asset":{"properties":{"asset:prop:name":"test-asset-2","asset:prop:version":"1.0","asset:prop:id":"test-asset-2"}},"dataAddress":{"properties":{"someprop":"someval","type":"test-dataaddress-2"},"type":"test-dataaddress-2"}},{"asset":{"properties":{"asset:prop:name":"test-asset-3","asset:prop:version":"1.0","asset:prop:id":"test-asset-3"}},"dataAddress":{"properties":{"someprop":"someval","type":"test-dataaddress-3"},"type":"test-dataaddress-3"}},{"asset":{"properties":{"asset:prop:name":"test-asset-4","asset:prop:version":"1.0","asset:prop:id":"test-asset-4"}},"dataAddress":{"properties":{"someprop":"someval","type":"test-dataaddress-4"},"type":"test-dataaddress-4"}},{"asset":{"properties":{"asset:prop:name":"test-asset-5","asset:prop:version":"1.0","asset:prop:id":"test-asset-5"}},"dataAddress":{"properties":{"someprop":"someval","type":"test-dataaddress-5"},"type":"test-dataaddress-5"}},{"asset":{"properties":{"asset:prop:name":"test-asset-6","asset:prop:version":"1.0","asset:prop:id":"test-asset-6"}},"dataAddress":{"properties":{"someprop":"someval","type":"test-dataaddress-6"},"type":"test-dataaddress-6"}},{"asset":{"properties":{"asset:prop:name":"test-asset-7","asset:prop:version":"1.0","asset:prop:id":"test-asset-7"}},"dataAddress":{"properties":{"someprop":"someval","type":"test-dataaddress-7"},"type":"test-dataaddress-7"}},{"asset":{"properties":{"asset:prop:name":"test-asset-8","asset:prop:version":"1.0","asset:prop:id":"test-asset-8"}},"dataAddress":{"properties":{"someprop":"someval","type":"test-dataaddress-8"},"type":"test-dataaddress-8"}},{"asset":{"properties":{"asset:prop:name":"test-asset-9","asset:prop:version":"1.0","asset:prop:id":"test-asset-9"}},"dataAddress":{"properties":{"someprop":"someval","type":"test-dataaddress-9"},"type":"test-dataaddress-9"}}]
```

Comment:
I just realize I never commented on these. Thanks for the thorough review. Since the points are all fixed, I'll set them to resolved. Please notify me if there's anything I missed.